include/usb/usb_device.h

```diff
@@ -38,11 +38,18 @@
 
 #include <drivers/usb/usb_dc.h>
```

Comment:
Either remove this header, or remove `#include <drivers/usb/usb_dc.h>` whenever you include this header.

Reply:
Done